integrations/activemq/prometheus_metadata.yaml

```diff
@@ -0,0 +1,56 @@
+platforms:
+  - type: GKE
+    launch_stage: GA
+    detections:
+      - characteristic_metric:
+          metric_type: prometheus.googleapis.com/activemq_connections_total/gauge
+    exporter_metadata:
+      name: ActiveMQ Prometheus Exporter
```

Comment:
Based on the URL, "JMX Exporter" feels like a more appropriate name for this exporter, unless the doc url is for the wrong exporter. The bitnami exporter also does not seem to have a version 0.17.0. Did we mean to use this instead: https://github.com/prometheus/jmx_exporter?

Reply:
I think "JMX Exporter" makes sense. A handful of integrations rely on it. I will update.

Bitnami packages `prometheus/jmx_exporter` into a container image, and it does have tag `0.17.0`. The Bitnami readme links to the Prometheus repo. We can switch to the Prometheus link if you think that makes more sense.

Reply:
We can keep the url as is if the bitnami jmx exporter is what we want users to be installing.